Syncing the github repository for s2s browsers with production

mrpatrickwest opened the pull request.

Commit:
Copied and updated production files in this branch

mrpatrickwest commented:
I Need to get this set up on udco before it can be merged.

zednis commented on opensearch/descriptions/datasets.xml:
```diff
@@ -13,7 +13,7 @@
   projects the datasets are related to), and the result set.</Description>
   <Tags></Tags>
-  <Contact>web@mail.deepcarbon.net</Contact>
+  <Contact>Han Wang</Contact>
```
We don't want to make this change.  Ditto for some of the other opensearch description document changes.

Commit:
dco specific widgets added to local directory

Commit:
Re-fixed the Contact and Developer elements

Commit:
The .htaccess tells apache that .ttl is mime type turtle file

Commit:
This one is shared, not dco specific, so removing

Commit:
using /browsers as the root of the dco s2s browsers

Commit:
Added  missing css and image files. Updated people.php to use relative
path to image instead of full URL. Updated dco-s2s.ttl to point to local
copies of local javaScript.

Commit:
Changed the vivo link to info.deepcarbon.net

Commit:
Merge pull request #7 from tetherless-world/devops-setup-docker

HTTPS & docker configuration

Commit:
Merge master into production-sync

Commit:
This is a shared widget in s2s. Not needed here.

Commit:
Merge branch 'production-sync' of https://github.com/tetherless-world/dco-s2s-browsers into production-sync

Conflicts:
	objects.html
	opensearch/descriptions/people.xml
	opensearch/services/field_studies/field_studies.php

Commit:
DynamicMonthYearDateWidget.js is shared in s2s. The other two already
exist in js/widgets.

Commit:
Boolean is local in js directory. Dynamic is in s2s

Commit:
A backup not needed

Commit:
Fixed problem with find -type f option. Added replace for udco

Commit:
Updated s2s to point to master
